apps/judicial-system/web/src/components/TagAppealState/TagAppealState.tsx

```diff
@@ -44,10 +49,19 @@ const TagAppealState: React.FC<React.PropsWithChildren<Props>> = ({
       }
     }
     if (state === CaseAppealState.RECEIVED) {
-      return {
-        color: 'darkerBlue',
-        text: formatMessage(tables.receivedTag),
-      }
+      if (
+        user?.institution?.type === InstitutionType.COURT_OF_APPEALS &&
+        !appealCaseNumber
```

Comment:
I prefer positive checks in if statements, i.e.

`user?.institution?.type === InstitutionType.COURT_OF_APPEALS && appealCaseNumber ? received : new`

What do you think?

Reply:
Actually on second thought, it makes more sense to keep it this way because this is the only specific case where we want to display the purple new tag. In all other cases we want to default to the second, also cases that aren't being viewed by InstitutionType.COURT_OF_APPEALS, so to make a positive check work I'd have to complicate the code more.